daemon/api_aspects.go

```diff
@@ -0,0 +1,118 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package daemon
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/snapcore/snapd/aspects"
+	"github.com/snapcore/snapd/overlord/auth"
+)
+
+var (
+	aspectsCmd = &Command{
+		Path:        "/v2/aspects",
+		GET:         getAspect,
+		POST:        setAspect,
+		ReadAccess:  openAccess{},
```

Comment:
Are the access policy finalised? Does this deserve mention at spec level?

Reply:
However, I think both accesses should be authenticatedAccess so I'll change that